src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+  let discount = 0;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content

Reply:
I give you an example:
`const PRISE = 40;
const SMALL_DISCOUNT = 20;
const SMALL_DISCOUNT_DAYS = 3;
...`
and so on

Reply:
if you have questions about that, feel free to ask in the chat